docs/data/material/components/breadcrumbs/breadcrumbs.md

```diff
@@ -66,10 +66,10 @@ The accessibility of this component relies on:
 - To prevent screen reader announcement of the visual separators between links, they are hidden with `aria-hidden`.
 - A nav element labeled with `aria-label` identifies the structure as a breadcrumb trail and makes it a navigation landmark so that it is easy to locate.
 
-## Experimental APIs
+## Toolpad
 
 ### Page Container
 
-The [PageContainer](https://mui.com/toolpad/core/react-page-container/) component in `@toolpad/core` is the ideal wrapper for the content of your dashboard. It makes the Material UI Container navigation aware and extends it with page title, breadcrumbs, actions, and more.
+The [PageContainer](https://mui.com/toolpad/core/react-page-container/) component in `@toolpad/core` is the ideal wrapper for the content of your dashboard. It makes the Material UI Container navigation-aware and extends it with page title, breadcrumbs, actions, and more.
 
 {{"demo": "./PageContainerBasic.js", "height": 400, "hideToolbar": true}}
```

Comment:
```suggestion
{{"demo": "PageContainerBasic.js", "height": 400, "hideToolbar": true}}
```

Reply:
Also, I don't think that we need the outline per of the nature of the component. That Paper elevation look pretty bad, we will likely change it, but I imagine this won't change the need for:
```suggestion
{{"demo": "PageContainerBasic.js", "height": 400, "hideToolbar": true, "bg": "inline"}}
```

Same as in https://github.com/mui/material-ui/pull/43748

Reply:
Also, the demo works in isolation so `hideToolbar": true` is wrong:

```suggestion
{{"demo": "PageContainerBasic.js", "height": 400, "bg": "inline", "defaultExpanded": false}}
```